graph/src/log/mod.rs

```diff
@@ -17,25 +28,24 @@ macro_rules! impl_slog_value {
     };
 }
 
-use isatty;
-use slog::*;
-use slog_async;
-use slog_envlogger;
-use slog_term::*;
-use std::{fmt, io, result};
-
-use crate::prelude::ENV_VARS;
-
 pub mod codes;
 pub mod elastic;
 pub mod factory;
 pub mod split;
 
-pub fn logger(show_debug: bool) -> Logger {
+pub fn logger(show_debug: bool, json: bool) -> Logger {
```

Comment:
I think this should take an Option<DrainPicker> if None, default to CustomFormat, if one passed then use that instead

Reply:
you can add a `json_logger` function which does that setup and then calls `logger(debug, drain)`

Reply:
could also setup an helper `default_logger` this way the other instances can just be replaced from `logger` to `default_logger` on tests and such